src/hooks/reviews/use-review.ts

```
@@ -0,0 +1,30 @@
+            try {
+                // Call the action to create a review
+                const res = await createReviewAction(data);
+
+                // Check if the response is valid
+                if (!res || res.error || res.error === "Failed") {
+                    throw new Error(res?.error || "Something went wrong");
+                }
+
+                // Return the response
+                return res;
+            } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)